client/shared/src/components/FileMatchChildren.tsx

```diff
@@ -105,7 +133,7 @@ export const FileMatchChildren: React.FunctionComponent<FileMatchProps> = props
                     </code>
                 </Link>
             ))}
-            {grouped.map(group => (
+            {grouped.map((group, groupIndex) => (
```

Comment:
`groupIndex` needed?
```suggestion
            {grouped.map((group) => (
```